packages/cli/src/index.ts

```diff
@@ -764,6 +777,66 @@ yargs
       commandWrapper(handlerBuildCommand)(argv as Arguments<BuildDocsArgv>);
     }
   )
+  .command(
+    'translations',
+    'Creates or updates translations.yaml files and fills them with missing built-in translations and translations from the redocly.yaml and sidebars.yaml files.',
+    (yargs) =>
+      yargs.options({
+        contentDir: {
+          alias: 'd',
+          type: 'string',
+          description: 'Project content directory.',
+          default: '.',
+        },
+        locale: {
+          alias: 'l',
+          type: 'string',
+          array: true,
+          description: 'Locale or a list of locales to generate translations for.',
+          required: true,
+        },
+        'lint-config': {
+          description: 'Severity level for config file linting.',
+          choices: ['warn', 'error', 'off'] as ReadonlyArray<RuleSeverity>,
+          default: 'warn' as RuleSeverity,
+        },
+      }),
+    (argv) => {
+      process.env.REDOCLY_CLI_COMMAND = 'translations';
+      commandWrapper(handleTranslations)(argv);
+    }
+  )
+  .command(
+    'eject <type> <name>',
+    'Ejects components.',
+    (yargs) =>
+      yargs
+        .positional('type', { required: true, choices: ['component'] })
+        .positional('name', { type: 'string', required: true })
+        .options({
+          contentDir: {
```

Comment:
I suggest something like `destination` or `destination-dir` as the name here.

Reply:
I'd also vote for adopting kebab-case version, but I'm not sure about the naming itself. 
@RomanHotsiy could you help us to decide on naming?

Reply:
I don't like naming this argument (contentDir) different in different commands.

I think we should name it same for develop/eject/translate etc.

contentDir is a path to the folder with the content of the portal. By default it uses current working dir but someone can change it like:

```
-d src/content
```

It's not widely used.

So the description here is invalid. It's not a folder to eject components into: if you provide `-d src/content` it won't eject into `src/content`, it will eject into `src/content/@theme/components/...`.

I am fine with kebab-case. We can also rename it to something else but I want to keep it consistent for all the commands.

We need to open a ticket to sync argument names with CLI and realm ASAP too.

Reply:
For `redocly preview` (what was the `develop` command in portal/realm package), we use `--source-dir` or `-d`. That command is already live, in case that influences anyone.

Reply:
Created the issue: https://github.com/Redocly/redocly/issues/10374
@RomanHotsiy @lornajane please help with finalising the names.

Reply:
@tatomyr, in the issue you created, you suggest renaming the parameter to `content-dir`. As @lornajane pointed out, we already use the name `--source-dir` in `redocly preview` and I agree that all the commands should use the same name for this property.

As for renaming the properties in Realm, we will do this soon but we won't be able to do it today. Let's use the current Realm interface for now.

Reply:
> For redocly preview (what was the develop command in portal/realm package), we use --source-dir or -d. That command is already live, in case that influences anyone.

So let's use `--source-dir` everywhere.
We can implement the change for realm in the next PR tomorrow. I'm pretty sure very few people if any use --source-dir at the moment. We can change it easily now and we won't be able to do it soon.

@BohdanPetryshyn how about renaming `develop` to `preview` in realm too? (we can support two until 1.0)

Reply:
`preview` vs `develop` is not as easy to change though as, in contrast to `--source-dir`, it's used by everyone.
I would suggest adding `preview` to the realm but keep using `develop` in CLI until the 1.0 of `realm` family.

Reply:
> We can change it [Realm] easily now and we won't be able to do it soon.

Our latest plan was to completely wrap all the realm/revel/reef/redoc CLI interactions with Redocly CLI and make the interface internal implementation detail. 

We already have an issue to remove any mentions of realm CLI from docs: https://github.com/Redocly/redocly/issues/10068. We should also ask the existing customers to use `redocly preview` in their scripts in package.json

@RomanHotsiy, what do you think? I'm trying to decrease the priority of this change in Realm as we have limited capacity at the moment.

Reply:
I understand that this is an implementation detail but why keep it unsynchronized if we can change it now? We won't be able to change it easily in the future.

I can pick it up if you are busy. Let me know.

Reply:
Prepared PR for that: https://github.com/Redocly/redocly/pull/10415.